### DISCUSSION TITLE
Slack is only for Puppet staffs ?

### DISCUSSION THREAD

**Discussion by hieutomra** [puppetlabs/community#86] [All Things Community]
I tried to join https://puppetcommunity.slack.com/ but it seems to require specific puppet email address ...

**Comment by davidsandilands** [comment]
Hi @hieutomra apologies for the issue here, I am working with IT and will get this resolved please use this link to join for now https://join.slack.com/t/puppetcommunity/shared_invite/zt-2v8f1qrmt-1aa6_M0itZUGMd4Fvk_VfA

**Comment by davidsandilands** [comment]
@hieutomra to confirm this should be resolved for you now?

**Comment by hieutomra** [comment]
yes I confirm that this now work. Thanks !
